### PR TITLE
repo: remind to git add only when using git

### LINKED ISSUE #1685: add: do not create `.gitignore` if `--no-scm` is specified
As discussed here with @robogeek https://github.com/iterative/dvc.org/pull/195#discussion_r262259472

Check `dvc add` and other commands.
Check `.gitignore` and other SCM related outputs

### PR DESCRIPTION
**Pull request by mroutis**
Fix #1685

### CODE REVIEW

File: dvc/repo/__init__.py [hunk 1]
```diff
@@ -93,7 +93,9 @@ def find_dvc_dir(root=None):
         return os.path.join(root_dir, Repo.DVC_DIR)
 
     def remind_to_git_add(self):
-        if not self.files_to_git_add:
+        is_git_repo = type(self.scm).__name__ == "Git"
```
Comment (pared) [inline dvc/repo/__init__.py]: Should we check it by name? Maybe something like:
`type(self.scm) is dvc.scm.Git`?

File: dvc/repo/__init__.py [hunk 2]
```diff
@@ -93,7 +93,9 @@ def find_dvc_dir(root=None):
         return os.path.join(root_dir, Repo.DVC_DIR)
 
     def remind_to_git_add(self):
```
Comment (efiop) [inline dvc/repo/__init__.py]: Also, @pared introduced `remind_to_track` method in https://github.com/iterative/dvc/pull/1673/files , so if scm is not git it simply won't do anything, so this change seems to be not needed anymore.

File: dvc/repo/__init__.py (hunk 1 above)
Comment (mroutis) [reply dvc/repo/__init__.py]: uff, great one, @pared :+1:

**Comment by mroutis** [comment]
Closing in favor of https://github.com/iterative/dvc/pull/1673

**Pull request closed without merging**